main/src/library/Fixpoint/Lowering.flix

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
We should probably find a different name for this phase (or for my phase for that matter).

Any suggestions?

Reply:
In traditional compiler terminology I believe "backend" is the closest match. If the interpreter was a stack-machine this phase would be part of the backend.

Reply:
VarsToIndices it is